tools/ml-audio-content-moderation/terraform/main.tf

```
@@ -0,0 +1,253 @@
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+provider "google" {
+  project = "${var.projectid}"
+}
+
+# Enable APIs; Must be individual resources or else it will disable all other APIs for the project.
+resource "google_project_service" "billingapi" {
+  service = "cloudbilling.googleapis.com"
+}
+
+resource "google_project_service" "schedulerapi" {
+  service = "cloudscheduler.googleapis.com"
+}
+
+resource "google_project_service" "pubsubapi" {
+  service = "pubsub.googleapis.com"
+}
+
+resource "google_project_service" "cfapi" {
+  service = "cloudfunctions.googleapis.com"
+}
+
+resource "google_project_service" "bqapi" {
+  service = "bigquery-json.googleapis.com"
+}
+
+resource "google_project_service" "gaeapi" {
+  service = "appengine.googleapis.com"
+}
+
+resource "google_project_service" "nlpapi" {
+  service = "language.googleapis.com"
+}
+
+resource "google_project_service" "speechapi" {
+  service = "speech.googleapis.com"
+}
+
+resource "google_project_service" "commentanalyzerapi" {
+  service = "commentanalyzer.googleapis.com"
+}
+
+resource "random_uuid" "uuid" {}
+
+
+# Create GCS Buckets
+# Create GCS Bucket to hold audio files on upload before they are processed by pipeline.
+resource "google_storage_bucket" "staging_audio_bucket" {
+  provider = "google"
+  name = "staging-audio-files-${substr(lower(random_uuid.uuid.result), 0, 20)}"
+}
+
+# Create GCS Bucket to hold audio files that have completed going through the pipeline.
+resource "google_storage_bucket" "processed_audio_bucket" {
+  provider = "google"
+  name = "processed-audio-files-${substr(lower(random_uuid.uuid.result), 0, 20)}"
+}
+
+# Create GCS Bucket to hold audio files that failed after going through STT API.
+resource "google_storage_bucket" "error_audio_bucket" {
+  provider = "google"
+  name = "error-audio-files-${substr(lower(random_uuid.uuid.result), 0, 20)}"
+}
+
+# Create GCS Bucket to hold transcription output files
+resource "google_storage_bucket" "transcription_bucket" {
+  provider = "google"
+  name = "transcription-files-${substr(lower(random_uuid.uuid.result), 0, 20)}"
+}
+
+# Create GCS Bucket to hold toxicity output files
+resource "google_storage_bucket" "output_bucket" {
+  provider = "google"
+  name = "output-files-${substr(lower(random_uuid.uuid.result), 0, 20)}"
+}
+
+
+# Create PubSub resources
+resource "google_pubsub_topic" "stt_topic" {
+  name = "${var.stt_queue_topic_name}"
```

You can drop the quotes (this applies everywhere):

```suggestion
  name = var.stt_queue_topic_name
```